src/E1_coldflow/config.h

```diff
@@ -69,11 +70,12 @@ namespace config {
     debug("Initializing sensors", DEBUG);
     sensors = new sensorInfo[numSensors];
     // sensorInfo s = ;
-    sensors[0] = {"Temperature",   FLIGHT_BRAIN_ADDR, 0, 3}; //&(testTempRead)}, //&(Thermocouple::readTemperatureData)},
+    sensors[0] = {"Lox PT Temperature",   FLIGHT_BRAIN_ADDR, 0, 3}; //&(testTempRead)}, //&(Thermocouple::readTemperatureData)},
     sensors[1] = {"All Pressure",  FLIGHT_BRAIN_ADDR, 1, 1};
     sensors[2] = {"Battery Stats", FLIGHT_BRAIN_ADDR, 2, 3};
     sensors[3] = {"Aux temp",      FLIGHT_BRAIN_ADDR, 4, 1};
```

Comment:
ooops - good catch...also on the excel sheet it says the clock cycles for aux temp is 4 but here it's 1